src/ploneintranet/network/browser/configure.zcml

```diff
@@ -30,4 +30,12 @@
       layer=".interfaces.IPloneIntranetNetworkLayer"
       />
 
+  <browser:page
+      name="getVocabulary"
```

Comment:
I'm slightly concerned about the generic name of this browser view, registered for *!